.github/actions/get-version/action.yml

```diff
@@ -0,0 +1,23 @@
+name: Return the version extracted from the branch name
+
+#
+# Returns the version from a branch name of a pull request. It expects the branch name to be in the format release/vX.Y.Z, release/X.Y.Z, release/vX.Y.Z-beta.N. etc.
```

Comment:
This won't work for repo's having multiple packages using different versions. Doesn't have to block merging this, but worth mentioning.

Reply:
👍 For sure. Some repos will need some extra customizing for individual characteristics and circumstances like that.